content/en/docs/ibm/create-cluster-vpc.md

```diff
@@ -0,0 +1,284 @@
+    r006-xxxxxxxx-dca9-4321-bb6c-960c4424xxxx   retrial-reversal-pelican-cavalier   10.240.64.0/18    us-south-2   false         true         2021-03-05T14:58:39+05:30   
+    r006-xxxxxxxx-7352-4a46-bfb1-fcbac6cbxxxx   subfloor-certainly-herbal-ajar      10.240.128.0/18   us-south-3   false         true         2021-03-05T14:58:39+05:30  
+    ```
+
+    - Now create a subnet as follows:
+
+    ```shell
+    ibmcloud is subnet-create my-subnet $VPC_ID $CLUSTER_ZONE --ipv4-cidr-block "10.240.128.0/18"
+    ```
+
+    Example output:
+    
+    ```
+    Creating subnet my-subnet in resource group kubeflow under account IBM as user new@user-email.com...
+                          
+    ID                  0737-27299d09-1d95-4a9d-a491-a6949axxxxxx   
+    Name                my-subnet
+    ```
+
+   - Make sure you export the subnet IDs follows:
+
+    ```shell
+    export SUBNET_ID=0737-27299d09-1d95-4a9d-a491-a6949axxxxxx
+    ```
+
+5. Create a `vpc-gen2` based Kubernetes cluster:
+    
+    ```shell
+    ibmcloud ks cluster create vpc-gen2 \
+    --name $CLUSTER_NAME \
+    --zone $CLUSTER_ZONE \
+    --version ${KUBERNETES_VERSION} \
+    --flavor ${WORKER_NODE_FLAVOR} \
+    --vpc-id ${VPC_ID} \
+    --subnet-id ${SUBNET_ID} \
+    --workers 2
+    ```
+
+6. Attach a public gateway
+
+   This step is mandatory for Kubeflow deployment to succeed, because pods need public internet access to download images.
+   
+   First check, if your cluster is already assigned a public gateway:
+   
+    ```shell
+    ibmcloud is pubgws
+    ```
+
+    Example output:
+
+    ```
+    Listing public gateways for generation 2 compute in all resource groups and region ...
+    ID                                          Name                                       Status      Floating IP      VPC                 Zone         Resource group   
+    r006-xxxxxxxx-5731-4ffe-bc51-1d9e5fxxxxxx   my-gateway                                 available   xxx.xxx.xxx.xxx       my-vpc              us-south-3   default   
+   
+    ```
+   
+   In the above run, gateway is already attached for the vpc: `my-vpc`. In case no gateway is attached, proceed with
+   rest of the setup.
+
+   
+   Attach a public gateway:
+   
+   ```shell
+   ibmcloud is public-gateway-create my-gateway $VPC_ID $CLUSTER_ZONE
+   ```
+
+   Example output:
+   ```
+   ID: r006-xxxxxxxx-5731-4ffe-bc51-1d9e5fxxxxxx
+   ```
+
+   Save the above generated gateway ID as follows:
+
+   ```shell
+   export GATEWAY_ID="r006-xxxxxxxx-5731-4ffe-bc51-1d9e5fxxxxxx"
+   ```
+
+   Finally, attach the public gateway to the subnet:
+   
+   ```
+   ibmcloud is subnet-update $SUBNET_ID --public-gateway-id $GATEWAY_ID
+   Updating subnet 0737-27299d09-1d95-4a9d-a491-a6949axxxxxx under account IBM as user new@user-email.com...
+                          
+   ID                  0737-27299d09-1d95-4a9d-a491-a6949axxxxxx   
+   Name                my-subnet   
+   ...
+   ```
```

Comment:
```suggestion
    ```shell
    ibmcloud is subnet-update $SUBNET_ID --public-gateway-id $GATEWAY_ID
    ```

    Example output:

    ```
    Updating subnet 0737-27299d09-1d95-4a9d-a491-a6949axxxxxx under account IBM as user new@user-email.com...
                          
    ID                  0737-27299d09-1d95-4a9d-a491-a6949axxxxxx   
    Name                my-subnet   
    ...
    ```
```

Reply:
(I think 4 spaces or 2 tabs (with 2 spaces each) will render better than 3 -- applied across the example, please double check)